README.md

```diff
@@ -115,6 +121,31 @@ file, they can be found in [`scripts`](./docker/package/scripts) and
 [`defaults`](./docker/package/defaults) folders respectively.
 
 
+### Multiple similar systemd services
+
+It's possible to run multiple same services, e.g. two `tezos-node`s that run different
+networks.
+
+In order to do that you should copy systemd service file and config file, e.g.:
+```
+cp /lib/systemd/system/tezos-node.service /lib/systemd/system/tezos-node-1.service
+cp /etc/default/tezos-node /etc/default/tezos-node-1
+```
+
+Adjust new `/etc/default/tezos-node-1` config file and point to `EnrironentFile` in
+`/lib/systemd/system/tezos-node-1.service` to it.
+After that, you should be able to run new service:
+```
+systemctl start tezos-node-1
+```
+
+Another case for running multiple similar systemd services is when one wants to have
+multiple daemons that target different protocols.
+Since daemons for different protocols are provided in the different packages, they will
```

Comment:
> Since daemons for different protocols are provided in the different packages

Is it true? Don't our latest packages have daemons for two protocols?

Reply:
no, we have separate packages for `-007-PsDELPH1` and for `-006-PsCARTHA`

Reply:
Ah, apparently we have package per executable, not one package for each release (1 for Ubuntu, 1 for Fedora), I forgot about it.